### PR TITLE
Create an introductory post for CMS Kit Demo application

### PR DESCRIPTION
**Pull request by EngincanV**
Resolves https://github.com/volosoft/vs-internal/issues/2836

---

I am not ensure about the title of the post (**"CMS Kit in Action: Unveiling ABP Framework's Content Management Module"**). 
I think we can find a more attractive one. Here are some alternatives:

* "Unleash the Power of ABP CMS Kit: A Dynamic Content Showcase”
* “Build your own dynamic content website with ABP CMS Kit”

I am open to suggestions.

### CODE REVIEW

**Review by braim23 — APPROVED** [review]

**Comment by braim23** [comment]
I suggest the following as the title:
"ABP's CMS Kit: Explore the Source Code of the Dynamic Content Masterpiece"

File: docs/en/Community-Articles/2023-09-23-CMS-Kit-Demo/POST.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+* **Reactions**: Allow users to react to your content using simple smileys
+* **Rating**: Reusable component to rate other contents
+* **Global Resources**: Dynamically add CSS / JavaScript to your pages or blog posts
+* **Dynamic Widgets**: Build widgets and use them in dynamic content, like blog posts
+
+> For more information, please check the [CMS Kit Module documentation](https://docs.abp.io/en/abp/latest/Modules/Cms-Kit/Index). In the documentation, you can see descriptions for each feature, learn how to install & configure the module, and much more...
+
+### CMS Kit Pro: The Building Blocks (a.k.a Features)
+
+The following features are provided by the CMS Kit Pro Module:
```
Comment (hikalkan) [inline docs/en/Community-Articles/2023-09-23-CMS-Kit-Demo/POST.md]:
```suggestion
CMS Kit Pro is a part of [ABP Commercial](https://commercial.abp.io/) and provides additional features. The following features are provided by the CMS Kit Pro Module: 
```

File: docs/en/Community-Articles/2023-09-23-CMS-Kit-Demo/POST.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+![](pages-admin-side.jpg)
+
+After you have created the page, you can access it via `/{slug}` URL on the public-web side:
+
+![](products-abp-commercial.png)
+
+## What's Next?
+
+Please try the CMS Kit Module now and provide [feedback](https://github.com/abpframework/abp) to help us to build a more effective content management kit!
```
Comment (hikalkan) [inline docs/en/Community-Articles/2023-09-23-CMS-Kit-Demo/POST.md]: We can add a resources section with the following links:

* demo source code link
* demo lşive link
* cms kit document link
* cms kit pro document link

**Review by hikalkan — CHANGES_REQUESTED** [review]

**Comment by hikalkan** [comment]
I liked "Unleash the Power of ABP CMS Kit: A Dynamic Content Showcase"

**Review by enisn — APPROVED** [review]

**Review by hikalkan — APPROVED** [review]

**Pull request merged**